code/src/components/ThoughtList.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable comma-dangle */
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable no-shadow */
+import React from 'react';
+import { formatDistance } from 'date-fns'; // did npm install date-fns
+
+const ThoughtList = ({ loading, thoughtList, onThoughtLikeChange }) => {
+  if (loading) {
+    return <h1 className="loading">😬 loading 😬</h1>
+  }
+
+  return (
+    <section className="thought-list-container">
+      {thoughtList.map((thought) => (
+        <div key={thought._id} className="thought-list">
+          <h4 className="the-thought">{thought.message}</h4>
+          <div className="thought-detail">
+            <button
+              type="button"
+              /* an if conditional that says if the count of the heart is zero,
+              then the className would be empty-button,
+              otherwise it would be like-button */
+              className={thought.hearts === 0 ? 'empty-button' : 'like-button'}
```

Comment:
It doesn't make much sense to show that the button is pressed if someone else liked the post, but the instructions weren't very clear about this and being able to like a post infinitely also makes no sense 🥴

Reply:
Haha how do you mean?